mend_import_sbom/import_sbom.py

```diff
@@ -575,16 +583,22 @@ def upload_to_mend(upload):
     ret = None
     try:
         json_prj = json.dumps(upload['projects'])  # API understands just JSON Array type, not simple List
-        upload_projects = [proj["coordinates"]["artifactId"] for proj in upload["projects"]]
+        upload_projects = [try_or_error(lambda: proj["coordinates"]["artifactId"], try_or_error(lambda: proj["projectToken"], "")) for proj in upload["projects"]]
         if len(upload_projects) > 1:
             proj_txt = "\n  ".join(upload_projects)
             logger.debug(f'[{fn()}] Uploading projects:\n  {proj_txt}')
         else:
             logger.debug(f'[{fn()}] Uploading project:  {upload_projects[0]}')
-
+        '''
         data = f"type=UPDATE&updateType={args.update_type}&agent={AGENT_INFO['agent']}&" \
                f"agentVersion={AGENT_INFO['agentVersion']}&token={args.ws_token}&userKey={args.ws_user_key}&" \
                f"product={args.ws_product}&timeStamp={ts}&diff={json_prj}"
+               
+               The different Agent names and versions are not acceptable for current API 1.4 method
+        '''
+        data = f"type=UPDATE&updateType={args.update_type}&agent=fs-agent&" \
+               f"agentVersion=''&token={args.ws_token}&userKey={args.ws_user_key}&" \
+               f"product={args.ws_product}&timeStamp={ts}&diff={json_prj}"
```

Comment:
This was originally accounted for in the code by not including ps- at the tool level. I think we still want this to show import-sbom.

Reply:
Unfortunately, when you create a request with an agent name  different to "fs-agent" - update-request is impossible to upload. Our UI gives an error. But in other places I use AGENT_INFO, just here removed it.

Reply:
It should work with "import-sbom" just not "ps-import-sbom" as it was working previously.